### DISCUSSION TITLE
Changing Color of GitHub Logo

### DISCUSSION THREAD

**Discussion by Compaszer** [community/community#112141] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,
I want to build a website that gives an overview of my projects. Each project should have a link to it's GitHub repo. For aesthetic reasons the link should be the GitHub logo.
On your page about permitted use of the GitHub Logo you suggest, that this is a permitted usecase. However you also state, that the color of the logo may not be changed, thus only a black or a white logo is available for permitted use.
Unfortunately this does not match the design and color language of my website, as all the other icons have a colored gradient that matches the theme color of my website. It would look very unpleasing if suddenly one icon would look completely different.

So my question is, if it is really not allowed to change the fill color of the logo to match the design language of the website.

**Comment by GitXpresso** [comment]
Maybe this will work https://imagecolorchanger.com don't like or link doesn't work search up image color changer and your find what you need, hope this helps!

**Reply by GitXpresso** [reply]
https://www.iconfinder.com/icons/291716/github_social_network_social_icon and click color editor but sign up if you don't have an account and once your done signing up and add your hex code almost forgot something click color editor and then click legacy editor to add a hex color to the background and image and you can reshape the ears of the git hub logo kinda like Photoshop but that's probably the only things in common figure and test the legacy editor out yourself, lookup git mocha on stack exchange I used the legacy editor to make my profile picture I'm going to find the hex of coffee cream which is E3CBA9 as the git hub logo and 6f4e27 coffee as the background and find an image of a coffee cup and add as the coffee in the cup, do you like that idea?

**Reply by kilva245** [reply]
thats nice

**Comment by queenofcorgis** [comment, marked as answer]
Per the [GitHub brandkit](https://brand.github.com/foundations/logo): The Invertocat and our wordmark should only appear in white, black, or in few cases grey.
